### ISSUE TITLE
Decide what to do with this repo.

### ISSUE THREAD

**Issue by samcunliffe** [UCL/dxss#69]
Seems we won't be working on this very much any more. So the question becomes: what should we do?

Perhaps "go dark" and make private? Perhaps archive the repo? Or delete the whole thing.

To my taste option 1 leaves the possibility that we could reinvigorate later if we decided we wanted to do that. But also it wouldn't be public so we don't accidentally get users for a thing that is incomplete?

**Comment by janoschpreuss** [comment]
Going private seems like a good option to me.

**Comment by matt-graham** [comment]
Agreed that making private is probably the best solution, will do so now.
